frontend/src/metabase/components/IconBorder/IconBorder.jsx

```diff
@@ -17,7 +17,7 @@ import CS from "metabase/css/core/index.css";
 
 export default class IconBorder extends Component {
   static propTypes = {
-    borderWidth: PropTypes.oneOf([PropTypes.string, PropTypes.number]),
+    borderWidth: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
This is completely unrelated and cleans up the dev tools console a bit by not displaying an error.